After update to 6.0.0-next.8 - Failed to compile.

Hello all,

I am using:

```
    "@angular/compiler": "~13.2.0",
    "@angular/core": "~13.2.0",
    "@angular/cli": "~13.2.1",

    "@ngx-formly/bootstrap": "^6.0.0-next.8",
    "@ngx-formly/core": "^6.0.0-next.8",
    "@ngx-formly/schematics": "^6.0.0-next.8",

    "@types/bootstrap": "^5.1.9",
    "bootstrap": "^5.1.3",
```

After updating from 6.0.0-next.7 to 6.0.0-next.8, I have the following errors:

```
Error: node_modules/@ngx-formly/core/lib/models/fieldconfig.d.ts:38:13 - error TS2411: Property 'minlength' of type 'string | ((error: any, field: FormlyFieldConfig) => string | Observable<string>) | undefined' is not assignable to 'string' index type 'string | ((error: any, field: FormlyFieldConfig) => string | Observable<string>)'.

38             minlength?: ValidationMessageOption['message'];
               ~~~~~~~~~
```


```
Error: node_modules/@ngx-formly/core/lib/models/fieldconfig.d.ts:40:13 - error TS2411: Property 'maxlength' of type 'string | ((error: any, field: FormlyFieldConfig) => string | Observable<string>) | undefined' is not assignable to 'string' index type 'string | ((error: any, field: FormlyFieldConfig) => string | Observable<string>)'.

40             maxlength?: ValidationMessageOption['message'];
               ~~~~~~~~~
```

I need help.

Thank you,
Iulius Ciorica

--- a/demo/src/app/ui/ui-primeng/datepicker/app.module.ts
+++ b/demo/src/app/ui/ui-primeng/datepicker/app.module.ts
@@ -3,7 +3,7 @@ import { CommonModule } from '@angular/common';
 import { ReactiveFormsModule } from '@angular/forms';
 import { FormlyModule } from '@ngx-formly/core';
 
-import { FormlyDatePickerModule } from '@ngx-formly/primeng/datepicker/';
+import { FormlyDatePickerModule } from '@ngx-formly/primeng/datepicker';
 
 import { AppComponent } from './app.component';
 

--- a/src/core/src/lib/models/fieldconfig.ts
+++ b/src/core/src/lib/models/fieldconfig.ts
@@ -40,13 +40,12 @@ export interface FormlyFieldConfig {
    * - `validation.show`: A boolean you as the developer can set to force displaying errors whatever the state of field. This is useful when you're trying to call the user's attention to some fields for some reason.
    */
   validation?: {
-    messages?: {
-      /** @deprecated use minLength */
+    messages?: { [messageProperties: string]: ValidationMessageOption['message'] } & {
+      /** @deprecated use `minLength` */
       minlength?: ValidationMessageOption['message'];
-      /** @deprecated use maxLength */
-      maxlength?: ValidationMessageOption['message'];
 
-      [messageProperties: string]: ValidationMessageOption['message'];
+      /** @deprecated use `maxLength` */
+      maxlength?: ValidationMessageOption['message'];
     };
     show?: boolean;
     [additionalProperties: string]: any;
@@ -201,8 +200,15 @@ export interface FormlyTemplateOptions {
   hidden?: boolean;
   max?: number;
   min?: number;
+
   minLength?: number;
+  /** @deprecated use `minLength` */
+  minlength?: number;
+
   maxLength?: number;
+  /** @deprecated use `minLength` */
+  maxlength?: number;
+
   pattern?: string | RegExp;
   required?: boolean;
   tabindex?: number;

